app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -3361,7 +3361,7 @@ export class Dispatcher {
       tip
     )
 
-    this.processMultiCommitOperationRebaseResult(
+    return this.processMultiCommitOperationRebaseResult(
```

Comment:
This one is for free 😆 

To make sure whoever `await`s for this function, is also `await`ing for `processMultiCommitOperationRebaseResult`. You can see how we do exactly this for `reorderCommits` too in https://github.com/desktop/desktop/pull/12429/files#diff-c8d5cceaa76fd0fe1490c522f46fa1c4b804fffe63ca13acaf0b272578a40b59R3258